doc/recording.md

```diff
@@ -0,0 +1,369 @@
+        --home-dir "/var/lib/rotter" \
+        --create-home \
+        --groups audio,jackuser \
+        --password '*' \
+        --system \
+        --shell /sbin/nologin \
+        --user-group \
+        rotter
+```
+
+#### ALSA device configuration and testing
+In case you want to use an [ALSA](http://www.alsa-project.org) device (such as
+an USB or PCI audio interface, or a virtual AoIP device) as your `jackd`
+backend, you have to configure it beforehand. Afterwards test that you're able
+to record from its capture device, with the help of `arecord`:
+```bash
+# Capture 10 seconds via arecord directly from the first ALSA device as the
+# rotter user
+su -l -s /bin/bash \
+   -c 'arecord -D hw:0 -c 2 -d 10 -r 48000 -f S32_LE -v /tmp/test-arecord.wav' \
+   rotter
+```
+
+Keep in mind, that you will have to change the jack configuration later on, if
+your device isn't available as `hw:0` (refer to `arecord -L`).  Also note, that
+you might have to adapt the sample format `-f ...` option according to your
+device's capabilities (refer to `arecord -D hw:0 --dump-hw-params`).
+
+If the recorded test file can be played as expected, continue with the next
+section.
+
+#### Jack
+Install `jackd` ([jack2](https://github.com/jackaudio/jack2)) and the helper
+tools
+([example-clients](https://github.com/jackaudio/jack2/tree/master/example-clients))
+from the packages provided by your distribution or compile it from source
+([jack2 releases](https://github.com/jackaudio/jack2/releases)).
+
+Allow headless operation of `jackd` by allowing users of the `rotter` group to
+own the first audio device service (you might have to adapt
+`org.freedesktop.ReserveDevice1.Audio0` according to your device):
+```bash
+wget -O /etc/dbus-1/system.d/rotter.conf \
+     https://raw.githubusercontent.com/radiorabe/centos-rpm-rotter/master/dbus-rotter.conf
+
+# Reload the D-Bus configuration
+systemctl reload dbus
+```
+
+Install the [jackd systemd service unit
+template](https://github.com/radiorabe/centos-rpm-rotter/blob/master/jackd%40.service)
+and [raar jackd systemd service instance
+override](config/systemd/jackd@raar.service.d/override.conf):
+```bash
+# Install the jackd system service unit template
+wget -O "/etc/systemd/system/jackd@.service" \
+     https://raw.githubusercontent.com/radiorabe/centos-rpm-rotter/master/jackd%40.service
+
+# Install the jackd@raar service instance unit override
+mkdir "/etc/systemd/system/jackd@raar.service.d"
+wget -O "/etc/systemd/system/jackd@raar.service.d/override.conf" \
+     https://raw.githubusercontent.com/radiorabe/raar/master/config/systemd/jackd%40raar.service.d/override.conf 
+
+# You might need to adapt the jackd service instance unit override to suite
+# your environment
+vi /etc/systemd/system/jackd@raar.service.d/override.conf
+
+# Reload systemd manager configuration
+systemctl daemon-reload
+```
+
+#### Rotter
+Install `rotter` from the packages provided by your distribution or [compile it
```

Comment:
I'm confused. Is it in the distro or are we providing RaBe APEL packages?

Reply:
This refers to other distributions (not CentOS 7).